test/net/sourceforge/kolmafia/FamiliarDataTest.java

```diff
@@ -154,4 +167,56 @@ public void canHandleGreyGooseCombatSkills() {
     assertFalse(KoLCharacter.hasCombatSkill(SkillPool.CONVERT_MATTER_TO_ENERGY));
     assertFalse(KoLCharacter.hasCombatSkill(SkillPool.CONVERT_MATTER_TO_POMADE));
   }
+
+  @Nested
+  class API {
+    @AfterEach
+    private void beforeEach() {
+      // Other tests add familiars to the character
+      // Start clean.
+      KoLCharacter.reset("");
+      KoLCharacter.reset("familiar data test");
+    }
+
+    @BeforeEach
```

Comment:
`@AfterAll`

Reply:
I want AfterEach - in case we ever put another api.php test in this nested class.
Surely AfterEach will run if there is only one test in the Nested class?
Thus also serving as AfterAll?

Reply:
I think it will, yes